scripts/globals/missions.lua

```diff
@@ -804,7 +804,7 @@ function getMissionMask(player)
             if (player:hasCompletedMission(WINDURST, tpz.mission.id.windurst.TO_EACH_HIS_OWN_RIGHT) == false and getMissionRankPoints(player, 10) == 1) then
                 first_mission = first_mission + 1024
             else
-                if (player:hasCompletedMission(WINDURST, tpz.mission.id.windurst.WRITTEN_IN_THE_STARS) == false and getMissionRankPoints(player, 11) == 1) then
+                if (player:hasCompletedMission(WINDURST, tpz.mission.id.windurst.WRITTEN_IN_THE_STARS) == true and getMissionRankPoints(player, 11) == 1) then
```

Comment:
I am pretty sure you don't need to add '== true' here. Lua assess the value is true and will continue. Also don't think you need the == 1 since the function getMissionRankPoints() already returns 1 or 0 which evaluates to true / false respectively. I could be wrong and I will let someone else correct me if I am. 

Nothing technically wrong here, just style things.

Reply:
> I am pretty sure you don't need to add '== true' here. Lua assess the value is true and will continue. Also don't think you need the == 1 since the function getMissionRankPoints() already returns 1 or 0 which evaluates to true / false respectively. I could be wrong and I will let someone else correct me if I am.
> 
> Nothing technically wrong here, just style things.

Regarding the ==true, you're correct of course. I preferred to write it this way just to be constant with other similar lines in the same files.
Regarding your second comment, I did not change the getMissionrankPoints() logic. I added a call to hasCompletedMission(), which returns a Boolean. I'm not very familiar with LUA to be honest so I wasn't even sure if casting Boolean to integer is allowed but I doubt it's not discouraged anyway.